deep_qa/data/dataset_readers/squad_sentence_selection_reader.py

```diff
@@ -39,52 +39,52 @@ def read_file(self, input_filepath):
         for article in tqdm(dataset):
             for paragraph in article['paragraphs']:
                 context_article = paragraph["context"]
-                # split the context_article into a list of sentences
+                # Split the context_article into a list of sentences.
                 sentences = nltk.sent_tokenize(context_article)
 
-                # make a dict from span indices to sentence. The end span is
+                # Make a dict from span indices to sentence. The end span is
                 # exclusive, and the start span is inclusive.
-                span_to_sentence_idx = {}
+                span_to_sentence_index = {}
                 current_index = 0
                 for sentence in sentences:
                     sentence_len = len(sentence)
-                    # need to add one to the end index to account for trailing
-                    # space after punctuation that is stripped by NLTK.
-                    span_to_sentence_idx[(current_index,
-                                          current_index + sentence_len + 1)] = sentence
+                    # Need to add one to the end index to account for the
+                    # trailing space after punctuation that is stripped by NLTK.
+                    span_to_sentence_index[(current_index,
+                                            current_index + sentence_len + 1)] = sentence
                     current_index += sentence_len + 1
                 for question_answer in paragraph['qas']:
-                    # shuffle the sentences
+                    # Shuffle the sentences.
                     random.shuffle(sentences)
                     question_text = question_answer["question"]
-                    answer_start_idx = question_answer["answers"][0]["answer_start"]
-                    # Get the full sentence corresponding to the answer
+                    answer_start_index = question_answer["answers"][0]["answer_start"]
+                    # Get the full sentence corresponding to the answer.
                     answer_sentence = None
-                    for span_tup in span_to_sentence_idx:
-                        start_span, end_span = span_tup
-                        if start_span <= answer_start_idx and answer_start_idx < end_span:
-                            answer_sentence = span_to_sentence_idx[span_tup]
+                    for span_tuple in span_to_sentence_index:
+                        start_span, end_span = span_tuple
+                        if start_span <= answer_start_index and answer_start_index < end_span:
+                            answer_sentence = span_to_sentence_index[span_tuple]
                             break
-                    if answer_sentence is None:
+                    else:
```

Comment:
I do think the `  # no break` comment is helpful for reading this, especially for people who aren't familiar with this python quirk.